src/test/java/com/wavefront/sdk/jersey/app/SampleApp.java

```diff
@@ -105,7 +105,7 @@ public int getFailureCount() {
       public void close() {
         // no-op
       }
-    })).build());
+    }, new ApplicationTags.Builder("myApp", "myService").build()).build()).build());
```

Comment:
Can you instantiate the application tags once and share it across the tracer and the wavefront jersey filter?